sandbox.html

```diff
@@ -531,12 +539,12 @@
 </template>
 <template data-action="closeMe">No Parameters needed.</template>
 
-<template id="common-layer-selection">
+<template id="common-object-ids">
   <div>
-    <label for="values">List of Object IDs *</label>
-    <textarea name="values" id="values" rows="5" required></textarea>
+    <label for="objectids">List of List of Object IDs *</label>
+    <textarea name="objectIds" id="objectids" rows="5" required></textarea>
     <small>
-      <p>A JSON value like <code>["object1","object2"]</code></p>
+      <p>A JSON value like <code>[["object1-id"],["object2-id"]]</code></p>
```

Comment:
But it worked with the simple array ...?! Since this API is not published, this is not a breaking change, but why making it less convenient for the default case in maps where we have only one ID value?

Reply:
Before the single array was send to the backend and one value was applied on all ids of the array of ids.... so the multi id selection was broken anyway

Reply:
:+1: